psi4/driver/endorsed_plugins.py

```diff
@@ -33,6 +33,11 @@
 except ImportError:
     pass
 
+try:
+    import gpu_dfcc
+except ImportError:
+    pass
```

Comment:
does gpu_dfcc have any python module dependencies that aren't already dependencies of psi, like `import scipy`? If so, add something like [this](https://github.com/loriab/psi4/blob/b8f18eceaa8b09af4b0d3527de5d47421fc14704/psi4/driver/endorsed_plugins.py#L42-L53).

Reply:
no, there aren't any additional python dependencies